### DISCUSSION TITLE
Missing information in logs in HA & Ring-Mqqt

### DISCUSSION THREAD

**Discussion by Tramatnois** [tsightler/ring-mqtt#841] [Support Q&A]
Hey folks, I’m brand new to this HA stuff. Maybe it’s a stupid question but I can’t find the information if an alarm is triggered which device triggered the alarm
E.g.
In the Ring App the event is documented like:
Ring alarm was triggered from rear entrance
But in HA RingMQTT it is displayed like this:
Ring alarm changed to  triggered. 

Did I miss something?
I really would like to see in HA which device triggered the alarm.

if this can’t be solved by standard I would appreciate if you could provide me another solution 😅

Best wishes from Germany

Stefan

**Comment by tsightler** [comment]
Ring-mqtt provides state for all the sensors, so you can use an automation to determine what sensors triggered the event if you want to have different actions based on what device triggered the alarm.  Unfortunately, there's no other way to know, please feel free to describe the use case as to why this would be useful, although it's unlikely this would be added any time soon.
